csrankings-j.csv

```diff
@@ -131,6 +131,7 @@ Jaehyuk Huh,KAIST,http://calab.kaist.ac.kr/~jhuh,NC4hxqEAAAAJ
 Jaeil Kim,Kyungpook National University,https://sites.google.com/view/ipalab,L8cMLmEAAAAJ
 Jaejin Lee,Seoul National University,http://aces.snu.ac.kr/~jlee,6JaKru0AAAAJ
 Jaejun Yoo,UNIST,https://sites.google.com/view/jaejunyoo,7NBlQw4AAAAJ
+Jaekyun Moon,KAIST,http://moonlab.kaist.ac.kr/people.html,dfH3YpkAAAAJ

```

Comment:
School of Electrical Engineering, not CS. Please clarify: must be able to _solely_ advise a PhD student for a PhD _in Computer Science_.

Reply:
Unfortunately, we don't have any homepage showing a courtesy appointment in CS. However, there are alumni of our lab who is currently working as a postdoctoral fellow in the Computer Science department at CMU, alumni who worked in School of Computer Science and Sofrware at Chung-Ang University (in Korea), and alumni who is currently working as an assistant professor in the Graduate School of AI at UNIST (in Korea). The page (http://moonlab.kaist.ac.kr/alumni.html) shows these facts, indirectly demonstrating that Prof. Jaekyun Moon can supervise Ph.D students in CS. In addition, his work is also more related to CS as can be seen from his recent publications (ICML, ICLR, NeurIPS, ECCV, CVPR, INFOCOM, BMVC etc.).

Reply:
Unfortunately, this is not sufficient. They must clearly meet the criteria described above.

Reply:
I'm sorry, but is there any way to be accepted even if there is no courtesy appointment from CS and no CS students in supervision? 

I couldn't find anywhere to inquire about this.

Thank you for your time and attention!

Reply:
Everyone listed in CSrankings must be full-time, tenure-track, and a potential supervisor for a PhD in Computer Science.